silence/silence.go

```diff
@@ -11,7 +11,7 @@ import (
 
 const createdBy = "OSD Cluster Readiness Job"
 
-type silenceRequest struct {
+type SilenceRequest struct {
```

Comment:
new export should have a description comment?

Reply:
Added a comment